db/migrate/20220905140827_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration[6.1]
+  def change
+    create_table :categories do |t|
+      t.string :name, uniq: true, null: false
```

Comment:
I don't think `uniq: true` is works. You need to add index
`add_index :categories, :name, unique: true`